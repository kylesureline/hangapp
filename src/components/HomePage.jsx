import React from 'react';
import { Link } from 'react-router-dom';

const HomePage = () => (
  <div className="box-layout">
    <div className="box-layout__box">
      <h1 className="box-layout__title">Hangapp</h1>
      <p>Test your knowledge!</p>
      <div className="input-group input-group--centered-column">
        <Link className="button input-group__item input-group__item--column" to="/about">About</Link>
        <Link className="button input-group__item input-group__item--column" to="/login">Login</Link>
        <Link className="button button--secondary input-group__item input-group__item--column" to="/signup">Signup</Link>
      </div>
    </div>
  </div>
);

export default HomePage;
