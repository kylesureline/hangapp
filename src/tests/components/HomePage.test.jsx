import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../../components/HomePage.jsx';

test('should render HomePage correctly', () => {
  const wrapper = shallow(<HomePage />);
  expect(wrapper).toMatchSnapshot();
});
