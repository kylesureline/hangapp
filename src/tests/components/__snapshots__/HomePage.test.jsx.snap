// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HomePage correctly 1`] = `
<div
  className="box-layout"
>
  <div
    className="box-layout__box"
  >
    <h1
      className="box-layout__title"
    >
      Hangapp
    </h1>
    <p>
      Test your knowledge!
    </p>
    <div
      className="input-group input-group--centered-column"
    >
      <Link
        className="button input-group__item input-group__item--column"
        to="/login"
      >
        Login
      </Link>
      <Link
        className="button button--secondary input-group__item input-group__item--column"
        to="/signup"
      >
        Signup
      </Link>
    </div>
  </div>
</div>
`;
